pyproject.toml

```diff
@@ -36,10 +36,12 @@ dev = [
     "seaborn>=0.9.0",
 ]
 jax = [
+    "bayeux>=0.1.6",
     "blackjax>=1.0.0",
     "jax>=0.3.1",
     "jaxlib>=0.3.1",
     "numpyro>=0.9.0",
+    "flowMC>=0.2.4",
```

Comment:
can remove unless we think this is particularly handy for user problems (I don't!)